plugins/action_generate_document/src/generateDoc.ts

```diff
@@ -32,6 +33,7 @@ const generateDoc: ActionPluginType = async ({
       applicationSerial,
       description,
       isOutputDoc,
+      toBeDeleted,
```

Comment:
On second thought, I think ideally we should have 'expiry' on file (timestamps field), which is set on creation, vs toBeDeleted